DEVELOPING.md

```diff
@@ -23,7 +25,7 @@ The command will produce the `revive` binary in the root of the project.
 
 To enable debug logging, set the `DEBUG` environment variable:
 
-```sh
+```shell
```

Comment:
While updating this fine, you can configure mdsf to support language_aliases

You can see an example here

https://github.com/go-fuego/fuego/blob/d30a1068246738617924057b99b47397d9cfc8ba/mdsf.json#L6

Reply:
You rolled back some of the changes with 6e6b81eee6b6a59234a694cecd9c04b7e2859c56

But you didn't update mdsf.json to configure the aliases yet, right?